bert_e/workflow/gitwaterflow/integration.py

```diff
@@ -136,6 +136,31 @@ def update(wbranch, source):
         prev = branch
 
 
+def check_integration_branches(job):
+    """Check if the integration branches can be created."""
+
+    approvals = set(job.pull_request.get_approvals())
+    if job.settings.approve:
+        approvals.add(job.pull_request.author)
+    approved_by_author = job.pull_request.author in approvals
+
+    always_create_integration = job.settings.always_create_integration_branches
+    create_integration = job.settings.create_integration_branches
+    always_create_integration_prs = job.settings.always_create_integration_pull_requests
+    create_prs = job.settings.create_pull_requests
+    multiple_dst_branches = len(job.git.cascade.dst_branches) <= 1
```

Comment:
You can go deeper and merge the following case into one variable. 

- `always_create_integration` with `create_integration`
- `always_create_integration_prs` and `create_prs`

Reply:
Done